doc/whats_new/v0.24.rst

```diff
@@ -230,12 +234,6 @@ Changelog
   :meth:`tree.DecisionTreeRegressor.fit`, and has not effect.
   :pr:`17614` by :user:`Juan Carlos Alfaro Jiménez <alfaro96>`.
 
-:mod:`sklearn.datasets`
```

Comment:
You should reintroduce these lines (233-238) at the exact same position.

Reply:
I have included entry for my change at lines (145-148). So shouldn't I include skleanrn.datasets entry at lines 237-242?

Reply:
These lines are red in the diff because you removed them (probably not on purpose but when fixing a conflict). Thus, you should reinsert such that we don't see any diff on GitHub (meaning that you did not edit the lines apart of your additional entry).